src/hosts/paths.rs

```diff
@@ -8,14 +8,15 @@ pub enum Browser {
     Firefox,
 }
 
-enum OsType {
+#[derive(Debug)]
```

Comment:
Can this be removed? (You might be more experienced than me with Rust, in which case - what's idiomatic?)

Reply:
I wouldn't say I'm *more* experienced, just *differently* experienced, perhaps... :)

Anyway, to my knowledge, it is idiomatic to derive or implement Debug if you are exporting a symbol.